projects/js-packages/charts/src/visx/group/index.ts

```diff
@@ -0,0 +1,5 @@
+// Re-export Group component from visx
```

Comment:
Is this a good idea?

Reply:
Seems reasonable to me. I like the way the resulting import makes it clear this is a visx component:

`import { Group } from '@automattic/charts/visx/group';`

Reply:
Does `export { Group } from '@visx/group';` work the same?

Reply:
Tracked with CHARTS-73